bouncerscript/test/__init__.py

```diff
@@ -18,10 +18,34 @@ def noop_sync(*args, **kwargs):
     pass
 
 
-async def return_true(*args):
+async def return_true_async(*args):
     return True
 
 
+def counted(f):
+    def wrapped(*args, **kwargs):
+        wrapped.calls += 1
+        return f(*args, **kwargs)
+    wrapped.calls = 0
+    return wrapped
+
+
+@counted
```

Comment:
@escapewindow Couldn't find a better way to hack the testing of this function. It's called twice in its functionality but I need it to return toggled boolean results when called subsequently. I'm open for feedback and change this if it's too ugly.